CHANGELOG.md

```diff
@@ -38,6 +38,9 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 * New `signed` revset function to filter for cryptographically signed commits.
 
+* `jj git push` now shows a hint to use `--deleted` if the only changed bookmarks
+  are deleted ones.
+
```

Comment:
nit: I don't think changelog entry is needed for stuff like this. The user doesn't have to know that a better hint will be displayed.

Reply:
I assumed it was worth mentioning because of prior art, e.g. https://github.com/jj-vcs/jj/blob/ee0e86fa63d8fa8165927f0f727cbb4d19568f3c/CHANGELOG.md?plain=1#L338-L339.

Reply:
It might be because the hint could be misleading. I personally wouldn't add changelog for `jj undo`, though.

Reply:
Misleading how?

Reply:
because reverting undone operation is valid? Anyway, I don't think we needed to add it to the changelog.